modulefiles/wcoss_cray

```diff
@@ -34,7 +34,6 @@ module load g2tmpl/1.10.0
 module load ip/3.3.3
 module load nemsio/2.5.2
 module load sfcio/1.4.1
-module load sigio/2.3.2
```

Comment:
@hertneky Removing sigio module makes executable building failed on WCOSS-Cray. It turns out there are more routines using sigio libray, e.g. GFSPOSTSIG.F. You might revert the line "module load sigio/2.3.2" in modulesfiles for a workaround. We will cleanup all sigio relevant code in another PR.  Please let me know if this plan is working for you.

Reply:
@WenMeng-NOAA I reverted back to loading the sigio module. This works for me and there are no issues.

Reply:
@hertneky Thanks!